docs/tutorials/projects/guardian.md

```diff
@@ -0,0 +1,850 @@
+
+   Replace the attributes JSON object with the following:
+
+   ```json
+   {
+   "source": "cam",
+   "pipeline": [
+       {
+       "type": "detections",
+       "attributes": {
+           "detector_name": "detector",
+           "confidence_threshold": 0.6
+       }
+       }
+   ]
+   }
+   ```
+
+   Click **Save config** in the bottom left corner of the screen.
+
+{{% /tab %}}
+
+{{% tab name="Raw JSON" %}}
+
+Next, on the [`Raw JSON` tab](/manage/configuration/#the-config-tab), replace the configuration with the following configuration which configures the [ML model service](/services/ml), the [vision service](/services/vision), and a [transform camera](/components/camera/transform):
+
+```json {class="line-numbers linkable-line-numbers" data-line="31-48,50-69"}
+{
+  "components": [
+    {
+      "name": "local",
+      "type": "board",
+      "model": "pi",
+      "attributes": {},
+      "depends_on": []
+    },
+    {
+      "name": "cam",
+      "type": "camera",
+      "model": "webcam",
+      "attributes": {
+        "video_path": "video0"
+      },
+      "depends_on": []
+    },
+    {
+      "name": "servo",
+      "type": "servo",
+      "model": "pi",
+      "attributes": {
+        "pin": "12",
+        "board": "local"
+      },
+      "depends_on": [
+        "local"
+      ]
+    },
+    {
+      "name": "transform_cam",
+      "type": "camera",
+      "model": "transform",
+      "attributes": {
+        "source": "cam",
+        "pipeline": [
+          {
+            "attributes": {
+              "detector_name": "detector",
+              "confidence_threshold": 0.6
+            },
+            "type": "detections"
+          }
+        ]
+      },
+      "depends_on": []
+    }
+  ],
+  "services": [
+    {
+      "name": "mlmodel",
+      "type": "mlmodel",
+      "model": "tflite_cpu",
+      "attributes": {
+        "model_path": "/home/pi/effdet0.tflite",
+        "label_path": "/home/pi/labels.txt",
+        "num_threads": 1
+      }
+    },
+    {
+      "name": "detector",
+      "type": "vision",
+      "model": "mlmodel",
+      "attributes": {
+        "mlmodel_name": "mlmodel"
+      }
+    }
+  ]
+}
+```
+
+Click **Save config** in the bottom left corner of the screen.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Navigate to your [robot's Control tab](/manage/fleet/robots/#control) to test the transform camera.
+Click on the transform camera panel and toggle the camera on, then point your camera at a person or pet to test if the vision service detects them.
+You should see a bounding box with label around different objects.
+
+<div style="max-width: 600px" >
+    <img src="../../img/guardian/test-transform.jpg" alt="the control tab transform camera panel">
+</div>
+
+## Program the Guardian
+
+With the guardian completely configured and the configuration tested, it's time to program the guardian to behave like a real guardian.
```

Comment:
```suggestion
With the guardian completely configured and the configuration tested, it's time to make the robot guardian behave like a "real" guardian.
Next, you'll use the [Viam Python SDK](/program/sdks) to program it to detect people and pets and scare them with music and LEDs. 
```
MAybe something like this-- think it would at least be helpful to spec that you're coding in Python and using an SDK to do so!

Reply:
With the guardian completely configured and the configuration tested, it's time to make the robot guardian behave like a "real" guardian by programming the person and pet detection, the lights, music, and the movement: